### ISSUE TITLE
Profile Function

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#22]
## User story
1. As a user
2. I want / need to be able to update my personal information and submit additional data in a profile section
3. So that I can keep my information up to date

## Acceptance criteria
* The profile tab should be accessible from every page of our web application
* User should be able to update information provided during registration in a profile tab 
* Profile tab should have it's own UI page
* User should be able to provide additional information related to functions in our system, such as friends or team members or a profile photo
* Before updating the information provided in the profile in the database, the database should make sure that the user remains unique in the database 
* This function should prevent SQL injection 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
